src/compiler/checker.ts

```diff
@@ -33089,8 +33093,7 @@ namespace ts {
             // Grammar checking for modifiers is done inside the function checkGrammarFunctionLikeDeclaration
             checkFunctionOrMethodDeclaration(node);
 
-            // Abstract methods cannot have an implementation.
-            // Extra checks are to avoid reporting multiple errors relating to the "abstractness" of the node.
+            // method signatures already report "implementation not allowed in ambient context" elsewhere

```

Comment:
This is the code I copied. While here, I deleted the first comment line because it restates the code itself, and reworded the second comment line because I couldn't figure out what it meant until I read the comments on the original PR.